src/main/java/io/netty/incubator/codec/http3/QpackDecoderAckEachStateSyncStrategy.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.incubator.codec.http3;
+
+/**
+ * An implementation of {@link QpackDecoderStateSyncStrategy} that sends an acknowledgment for each
+ * {@link #entryAdded(long)}.
+ */
+final class QpackDecoderAckEachStateSyncStrategy implements QpackDecoderStateSyncStrategy {
+    static final QpackDecoderAckEachStateSyncStrategy ACK_EACH_STATE_SYNC_STRATEGY =
```

Comment:
consider removing this class and just expose it directly in `QpackDecoderStateSyncStrategy` (which is public).